lib/RadioSmileButton.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'SmileButton.dart';
+
+class RadioSmileButton extends StatefulWidget {
+  final int count;
+  final double buttonSize;
+  final Color filColor;
+  final Color activeColor;
+  final Color accentColor;
+  final bool isEnabled;
+  final List<String> titles;
+  final List<String> imagesPaths;
+
+  final void Function(bool state, int index) onChanged;
+
+  const RadioSmileButton({
+    Key key,
+    this.buttonSize,
+    this.titles,
+    this.filColor,
+    this.activeColor,
+    this.accentColor,
+    this.isEnabled,
+    this.imagesPaths,
+    this.onChanged,
+    this.count,
+  }) : super(key: key);
+
+  _RadioSmileButtonState createState() {
+    return _RadioSmileButtonState();
+  }
+}
+
+class _RadioSmileButtonState extends State<RadioSmileButton> {
+  bool _canChanged = true;
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: _radioButtonsBuild(),
+    );
+  }
+
+  List<Widget> _radioButtonsBuild() {
+    List<Widget> buttons = [];
+    List<GlobalKey> _buttonsKeys = [];
+
+    if(widget.count == widget.titles.length && widget.count == widget.imagesPaths.length) {
+      for (int i = 0; i < widget.count; i++) {
+        GlobalKey _buttonKey = new GlobalKey();
+        print(_buttonKey);
```

Comment:
If you use print, at least add some description to the parameter,
for example
        print("_RadioSmileButtonState, _radioButtonsBuild, _buttonKey= $_buttonKey");